### PR TITLE
Add WSL2 Installation Instructions to the README file

### LINKED ISSUE #9544: Add WSL2 Installation Instructions to the README file
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [README.md](https://github.com/publiclab/plots2/blob/main/README.md) in the `plot2` repository (press the little pen Icon) and edit the line as shown below: 
1.  Go to line 46, press enter :
2.  Copy paste  the following:
      - [Windows Subsystem for Linux 2 Installation](#Windows Subsystem for Linux 2 Installation)
3.  Jump to line 124, right after the Windows Installation instructions, press enter.
4. Copy paste  the following:
      ## Windows Subsystem for Linux 2 Installation
      ####  Open the WSL 2 Terminal and type out the commands below
     1.  Install Dependencies required from Ruby Source with the following commands:
         -   *sudo apt update*
         -   *sudo apt install curl g++ gcc autoconf automake bison libc6-dev*
         -   *sudo apt install libffi-dev libgdbm-dev libncurses5-dev libsqlite3-dev libtool*
         -  *sudo apt install  libyaml-dev make pkg-config sqlite3 zlib1g-dev libgmp-dev*
         -   *sudo apt install  libreadline-dev libssl-dev*
         
     2.  Add GPG Key & Install RVM:
            - [ ]  gpg --keyserver hkp://keys.gnupg.net --recv-keys 
              409B6B1796C275462A1703113804BB82D39DC0E37D2BAF1CF37B13E2069D6956105BD0E739499BDB
           - [ ]  curl -sSL https://get.rvm.io | bash -s stable
       
     3.  Load the Script environment variables using source command:
           -   *source ~/.rvm/scripts/rvm*
     4.   Install  and use specific Ruby version:
           -   *rvm install ruby-x.x.x (this projects Ruby version)*
           -   *rvm --default use ruby-x.x.x (replace x.x.x with this project's Ruby version)*
     5.   For further reference, [read here ](https://linuxize.com/post/how-to-install-ruby-on-ubuntu-20-04/)
          
- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming the first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by sobekcore**
Fixes #9544

Adds detailed guidelines on how to install the app on Windows Subsystem for Linux, as well as its own marker in the Table of Contents.

### CODE REVIEW

**Review by Elukoye — COMMENTED** [review]
@sobekcore ,good job 🙂.LGTM 👌.
Please tag @RuthNjeri or @jywarren for a second review,approval and merging.
As you wait for that ,you can create an Fto issue to help  other first timers or look for another issue 🙂🚀.

**Comment by sobekcore** [comment]
Thanks! Also @jywarren could you review, approve and merge my PR?

File: README.md [hunk 1]
```diff
@@ -121,6 +122,26 @@ We recommend you either work in a virtual environment, or on a dual booted syste
 2. [Dual Booting](https://www.tecmint.com/install-ubuntu-alongside-with-windows-dual-boot/amp/), [option2](https://askubuntu.com/questions/1031993/how-to-install-ubuntu-18-04-alongside-windows-10), [video guide](https://www.youtube.com/watch?v=qNeJvujdB-0&fbclid=IwAR0APhs89jlNR_ENKbSwrp6TI6P-wxlx-a0My9XBvPNAfwtADZaAXqcKtP4)
 3. [Setting up a Linux virtual env](https://itsfoss.com/install-linux-in-virtualbox/)
 
+## Windows Subsystem for Linux 2 Installation
+
+**Open the WSL 2 Terminal and type out the commands below**
```
Comment (jywarren) [inline README.md:127]: This looks great!!! Should we say something a little more full like, 

```suggestion
Before continuing with the installation steps in this README, users of Windows Subsystem for Linux 2 (WSL 2) should open the WSL 2 Terminal and type out the commands below.
```

What do you think, @Elukoye?

@sobekcore you should be able to say "accept suggestion" if this looks good to you! Thanks again!

**Review by jywarren — CHANGES_REQUESTED** [review]

**Comment by jywarren** [comment]
This is so great. Shall we also add a link to #3840 for people working on it in there?

**Comment by sobekcore** [comment]
I've applied the changes that you suggested, i think it looks great now!

File: README.md (hunk 1 above)
Comment (Elukoye) [reply README.md]: Accepted suggestions 🙂👌

Comment (Elukoye) [reply README.md]: @jywarren ,@sobekcore

**Comment by Elukoye** [comment]
@jywarren  ,I think it's ok to merge it .@sobekcore  great job.🙂

**Review by cesswairimu — APPROVED** [review]
Looks great :tada:

**Comment by cesswairimu** [comment]
thanks for working on this @sobekcore :tada:  and thanks @Elukoye  and @jywarren for the reviews :tada:

**Comment by jywarren** [comment]
And merged! Great work everyone!!!

**Pull request merged**